.github/workflows/test_pr_changes.yml

```diff
@@ -26,41 +26,21 @@ jobs:
     runs-on: ubuntu-latest
     # Set job outputs to values from filter step
     outputs:
-      wire-ios: ${{ steps.filter.outputs.wire-ios == 'true' || steps.filter.outputs.carthage == 'true' }}
-      wire-ui: ${{ steps.filter.outputs.wire-ui }}
-      wire-ios-sync-engine: ${{ steps.filter.outputs.wire-ios-sync-engine }}
-      wire-ios-data-model: ${{ steps.filter.outputs.wire-ios-data-model }}
-      wire-ios-system: ${{ steps.filter.outputs.wire-ios-system }}
-      wire-ios-request-strategy: ${{ steps.filter.outputs.wire-ios-request-strategy }}
-      wire-api: ${{ steps.filter.outputs.wire-api }}
-      wire-analytics: ${{ steps.filter.outputs.wire-analytics }}
-      wire-domain-project: ${{ steps.filter.outputs.wire-domain-project }}
-      wire-domain: ${{ steps.filter.outputs.wire-domain }}
-      wire-ios-transport: ${{ steps.filter.outputs.wire-ios-transport }}
-      wire-ios-share-engine: ${{ steps.filter.outputs.wire-ios-share-engine }}
-      wire-ios-cryptobox: ${{ steps.filter.outputs.wire-ios-cryptobox }}
-      wire-ios-mocktransport: ${{ steps.filter.outputs.wire-ios-mocktransport }}
-      wire-ios-notification-engine: ${{ steps.filter.outputs.wire-ios-notification-engine }}
-      wire-ios-protos: ${{ steps.filter.outputs.wire-ios-protos }}
-      wire-ios-images: ${{ steps.filter.outputs.wire-ios-images }}
-      wire-ios-link-preview : ${{ steps.filter.outputs.wire-ios-link-preview }}
-      wire-ios-utilities: ${{ steps.filter.outputs.wire-ios-utilities }}
-      wire-ios-testing: ${{ steps.filter.outputs.wire-ios-testing }}
-      wire-foundation: ${{ steps.filter.outputs.wire-foundation }}
-      scripts: ${{ steps.filter.outputs.scripts }}
+      folders: ${{ steps.filter.outputs.changes }}
 
     steps:
       - uses: actions/checkout@v4
         with:
-          lfs: 'true'
+          lfs: 'false' # as long as we use lfs only for snapshotTesting it should not be useful here
```

Comment:
I don't understand this change. Don't we run snapshot tests using this workflow?

Reply:
@samwyndham this part of the job only detect changes, so we don't need the references here. the _reusable_run_tests.yml will run them with lfs.

You could argue that we miss trigger tests if only references change, but i don't think it's a common use case, it will always involve some code change